docs/proposal-rose-suite-run.md

```diff
@@ -39,66 +39,237 @@ The remaining points can happen any time after 1) is complete although note that
+  installed workflow.
+  * Note that this will have the effect of removing any files installed via
+    `rose-suite.conf`. They will get recreated by a subsequent reload or play but
+    could affect a running workflow.
+* Any options specified as part of the original install will be retained unless
+  overridden on the command line.
+* Consider a `--dry-run` option to report what would be changed?
+  * Does this replace `rose suite-cmp-vc`?
+
+`cylc reload` - no change from existing command.
+
+`cylc validate` - as per existing command with the following changes:
+* If validating a workflow installed with `cylc install` then use any relevant
+  command line options specified as part of the install unless overridden on the
+  command line.
+* Will work fine on source directories which include `rose-suite.conf` files.
+* No `--strict` mode
+  (naked dummy tasks no longer allowed - all tasks must be declared).
+  * This one is controversial and requires further discussion!
+* Cyclic graph validation can be quite slow and is currently included as part of
+  `--strict`. Add a `--no-cyclic-graph-validation` option to disable this?
+
+Several combinations of the above commands will be commonly used.
+Propose to support these as separate commands:
+* `cylc install-play` (`cylc ip` for short?)
+* `cylc install-validate`
+* `cylc reinstall-reload`
+
+`cylc pause|unpause` - new commands replacing `cylc hold|release|unhold`
+* Note that when you use `cylc stop` the scheduler will be modified to unpause
+  the workflow before stopping.
+  This means that a stopped workflow will not be paused when it is continued via
+  `cylc play`. On the other, a workflow which is continued for any other reason
+  (e.g. the server died) will retain its paused status.
+
+`cylc register` - remove: functionality no longer required.
+* `cylc install` will now perform the task of associating the source and run
+  directories via a symlink.
+* `cylc play` has no need to do this since the workflow definition will now
+  always be found in the run directory.
+
+`cylc clean` - new command to replace `rose suite-clean`.
+* This should support options which allow you to just clean subsets of the data.
+  For example, if you clean the `log` directory this allows you to rerun a
+  workflow (keeping the existing data in `share` if that is what you want).
+  The default will be to remove everything including the top level directory.
+* Support an option to clean all install targets other than localhost?
+* We may also want an option to archive the log directory rather than remove it.
+  This could simply imply renaming the directory, perhaps with a timestamp
+  (similar to what `rose suite-run` does but without tar gzipping the
+  directory). This would allow a workflow to be warm started without completely
+  losing the previous log files (this is an operational requirement at present
+  although we hope that re-flows will reduce or remove the need for this).
```

Comment:
Ooh, a use case for reflows :boom:

Reply:
And quite a nice one too as warm starts aren't the nicest. This would require the ability to stop the previous flow.

Reply:
We have that ability: `cylc stop --flow=LABEL` :boom:

Reply:
https://github.com/cylc/cylc-flow/issues/3864#issuecomment-708699402